addons/product/views/product_views.xml

```diff
@@ -676,7 +676,9 @@ action = {
                                         groups="product.group_product_variant"
                                         options="{'color_field': 'color'}"/>
                             </div>
-                            <field name="image_128" widget="image" alt="Product" invisible="not image_128" class="ms-auto" options="{'size': [55, 55]}"/>
+                            <aside class="ms-auto">
```

Comment:
we cannot do that in stable

Reply:
> we cannot do that in stable

yes. we will do this in the master version, so we don't have to add custom scss file.